File: mysdk/src/main/java/com/joker/mysdk/base/BaseApplication.java
package com.joker.mysdk.base;

import android.app.Application;


/**
 * Created by tv on 2018/7/2.
 */

public class BaseApplication extends Application {
    public static BaseApplication application;
    @Override
    public void onCreate() {
        super.onCreate();
        application = this;
    }

    public static Application getApplication() {
        return application;
    }

}

File: app/src/main/java/com/joker/zahuo/base/BaseApplication.kt
package com.joker.zahuo.base

import android.app.Application
import com.joker.zahuo.utils.SophixUtils

/**
 * Created by PengXJ on 2018/8/28.
 */
class BaseApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        SophixUtils.initSophix(this)
    }
}